### PR TITLE
AWS SDN FIPS workflow

### PR DESCRIPTION
**Pull request by lenahorsley**
[PERFSCALE-2617](https://issues.redhat.com/browse/PERFSCALE-2617)

### CODE REVIEW

**Comment by lenahorsley** [comment]
@paigerube14 
@svetsa-rh 
@qiliRedHat

File: ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+../../OWNERS
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS:1]: something is wrong with the owners file here, be sure theres an owners file at each folder you added

File: ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/openshift-qe-installer-aws-sdn-fips-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-qe-installer-aws-sdn-fips
+  steps:
+    env:
+      FIPS_ENABLED: "true"
+    pre:
+      - chain: ipi-aws-sdn-pre
+      - chain: create-infra-move-ingress-monitoring-registry
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/openshift-qe-installer-aws-sdn-fips-workflow.yaml:8]: let's move this step to be the last pre step

File: ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS (hunk 1 above)
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS:1]: this owners file should refernce back to the OWNERS file [here](https://github.com/openshift/release/blob/af37b5aa8a09837fd5d65208255caa619149442d/ci-operator/step-registry/openshift-qe/OWNERS) 

Think it needs 4 "../"

File: ci-operator/step-registry/openshift-qe/installer/aws/sdn/OWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+../../OWNERS
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/aws/sdn/OWNERS:1]: this owners file should refernce back to the OWNERS file here

Think it needs 3 "../"

File: ci-operator/step-registry/openshift-qe/installer/aws/sdn/OWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+../../../../OWNERS
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/aws/sdn/OWNERS:1]: think you have too many ../ here

**Comment by lenahorsley** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-sdn-fips

**Comment by lenahorsley** [comment]
@AlexNPavel 
@bradmwilliams 
@liangxia 
@hoxhaeris

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**